content/en/monitors/incident_management/incident_settings.md

```diff
@@ -79,6 +79,29 @@ In addition to the five default fields and the fields based on metric tags, you
 
 *Single-Select*, *Multi-Select*, and *Number* custom fields are searchable facets in the [Incident Homepage][4] and [Incident Management Analytics][2] for easy filtering of incidents. *Number* fields are measures in Incident Management Analytics that can be graphed and visualized in [Dashboards][13] and [Notebooks][11].
 
+### Responder types
+
+<div class="alert alert-warning">
+This feature is in open beta.
+</div>
+
+{{< img src="monitors/incidents/responder_types_settings.png" alt="The settings section dedicated to creating custom responder types" style="width:80%;">}}
+
+The responder types settings provide you with the ability to create custom roles to [assign to your incident responders][16] and to specify if those roles are meant to be held by one person or multiple people per incident. These roles are unrelated to the [Role Based Access Control (RBAC)][15] system. Responder types allow your responders to understand what their responsibilities are in an incident based on the definitions of your own incident response process. By default there are two roles:
+
+1. `Incident Commander` - The individual responsible for leading the response team 
+2. `Responder` - An individual that actively contributes to investigating an incident and resolving its underlying issue
+
+**Note:** Only `Incident Commander` appears in the Incident Settings so that you may customize its description. `Incident Commander` cannot be deleted as a responder type, nor can its name or status as a `One person role` be changed. The `Responder` role does not appear in the Incident Settings at all, due to it being a generic fallback role if a responder is not otherwise assigned a different role.
```

Comment:
```suggestion
**Note:** The `Incident Commander` responder type appears in Incident Settings so that you may customize its description. `Incident Commander` cannot be deleted as a responder type, nor can its name or status as a `One person role` be changed. The `Responder` role is a generic fallback role if a responder is not otherwise assigned a different role, and does not appear in Incident Settings.
```